### PR TITLE
Wicket 7 conversion

### PR DESCRIPTION
**Pull request by bradh**
POM conversion per advice at https://cwiki.apache.org/confluence/display/WICKET/Migration+to+Wicket+1.5#MigrationtoWicket1.5-Mavenusersarerecommendedtodependonorg.apache.wicket:wicket:pomfromnowon

Convert over usage of AbstractValidator to IValidator. Those are untested, but do at least fix compilation problems.

### CODE REVIEW

File: src/web/core/src/main/java/org/geoserver/web/data/store/WMSStoreNewPage.java [hunk 1]
```diff
@@ -8,14 +8,15 @@
 import java.io.IOException;
```
Comment (aaime) [inline src/web/core/src/main/java/org/geoserver/web/data/store/WMSStoreNewPage.java]: Let's not forget to update copyright headers. With 2016 now :-)

**Comment by aaime** [comment]
Thanks, looks good. About the headers hum... unsure, I guess we are going to modify all those files once more again anyways, we might just fix the headers during the sprint.

File: src/web/core/src/main/java/org/geoserver/web/data/store/WMSStoreNewPage.java (hunk 1 above)
Comment (bradh) [inline src/web/core/src/main/java/org/geoserver/web/data/store/WMSStoreNewPage.java]: I'm sure I did those changes last year...

Comment (aaime) [inline src/web/core/src/main/java/org/geoserver/web/data/store/WMSStoreNewPage.java]: Yeah, this file still states only 2014. Anyways, as said, no big deal, it's quite likely that we are going to have to modify these files further during the sprint.

**Comment by aaime** [comment]
Merged :)

**Pull request merged**